NHCallOrchestrator/handler.ts

```diff
@@ -0,0 +1,52 @@
+import * as t from "io-ts";
```

Comment:
the function (file name) should describe the operation (ie. HandleNotificationCalls, a verb)